src/components/ThreadEnvelope.vue

```diff
@@ -60,6 +61,9 @@
 				@click.native.prevent="$emit('toggle-expand', $event)">
 				<div class="sender">
 					{{ envelope.from && envelope.from[0] ? envelope.from[0].label : '' }}
+					<p class="sender__email">
+						{{ envelope.from[0].email }}
```

Comment:
like on L63 we should save-guard a possibly missing sender. messages might not have a `from` address